### ISSUE TITLE
Rubin isn't a telescope

### ISSUE THREAD

**Issue by timj** [astropy/astropy-data#139]
### Description

The telescope data file sites.json currently lists rubin and rubin_aux. The 8.4m telescope is called the Simonyi Survey Telescope and is part of Vera C. Rubin Observatory so "rubin" might not be the right name as described. LSST is not a telescope so the aliases shouldn't really include it. AuxTel is formally the Rubin Auxiliary Telescope. It's 1.2m (not 1.4m: see https://noirlab.edu/public/programs/vera-c-rubin-observatory/rubin-auxtel/) and should not include LSST in the aliases either.

We've just got the AAS facilities list updated to use Rubin:Simonyi and Rubin:1.2m.

### Expected behavior

I'm not sure if it's even possible to change "Rubin" without breaking people's code -- is it possible to deprecate sites?

I'm open for ideas as to how best to proceed with this.

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

```python
# Put your Python code snippet here.
```

**Comment by pllim** [comment]
Would putting the old name in "aliases" enough? I actually don't know if that would work as intended or not. Maybe @hamogu or @adrn know.

https://github.com/astropy/astropy-data/blob/5fe4a2d3d82407ac4672f288d9643aa60e54752f/coordinates/sites.json#L534

**Comment by hamogu** [comment]
That's exactly what the alias is for: Previous names is something is renamed, names that are not really the official names, common abbreviations etc.
Note however, that the file is called "sites.json", not "legal telescope names.json". That means that an "observatory site" might in be there just as well as a "telescope site". In practice, people say "I observed at Magellan ", not "I observed with Baade".

So, I would suggest to out the correct names as primary and leave LSST and Rubin as aliases.
